cirq/circuits/optimization_pass_test.py

```diff
@@ -149,12 +155,17 @@ def test_point_optimizer_raises_on_gates_changing_qubits():
     class EverythingIs42(cirq.PointOptimizer):
         """Changes all single qubit operations to act on LineQubit(42)"""
 
-        def optimization_at(self, circuit, index, op):
+        def optimization_at(
+            self, circuit: Circuit, index: int, op: 'cirq.Operation'
+        ) -> Optional[PointOptimizationSummary]:
             if len(op.qubits) == 1:
+                op = cast(cirq.GateOperation, op)
```

Comment:
Is this `cast` required? Maybe add `isinstance(op, cirq.GateOperation)` to the if condition instead, since this test doesn't appear to have behavior for non-gate operations.

Reply:
Done.